discretize/TreeMesh.py

```diff
@@ -441,90 +462,102 @@ def _pointer(self, index):
     def __contains__(self, v):
         return self._asIndex(v) in self._cells
 
-    def refine(
-        self, function=None, recursive=True, cells=None, balance=True,
```

Comment:
this is also fine for pep8 

(personally, I find this a bit easier to read than the below - it is only one indentation level rather than having different levels throughout the code base). 
thoughts @fourndo, @rowanc1 ?